pyccl/tracers.py

```diff
@@ -188,33 +190,46 @@ def get_kernel(self, chi):
         in this `Tracer`.
 
         Args:
-            chi (float or array_like): values of the comoving
-                radial distance in increasing order and in Mpc.
+            chi (float or array_like, optional): values of the comoving
+                radial distance in increasing order and in Mpc. If None,
+                returns the kernel at the internal spline knots.
 
         Returns:
-            array_like: list of radial kernels for each tracer. \
-                The shape will be `(n_tracer, chi.size)`, where \
-                `n_tracer` is the number of tracers. The last \
-                dimension will be squeezed if the input is a \
+            array_like: list of radial kernels for each tracer.
+                The shape will be `(n_tracer, chi.size)`, where
+                `n_tracer` is the number of tracers. The last
+                dimension will be squeezed if the input is a
                 scalar.
+                If no chi was provided, returns two arrays, the radial kernels
+                and the comoving radial distances.
```

Comment:
"... corresponding to the internal values used for interpolation." or something like that?

Reply:
Done.